src/shared/futureView/MembersList/MembersList.style.tsx

```diff
@@ -0,0 +1,69 @@
+import { withStyles, Theme, WithStyles } from 'shared/styles';
+import { rule } from 'shared/helpers/style';
+import { theme as extraTheme } from 'shared/styles/theme';
+
+const bottomBorder = `0px 1px 0px rgba(0, 0, 0, 0.1)`;
+
+const horizontalCellPadding = extraTheme.spacing.unit * 1.5;
```

Comment:
Calculate the `horizontalCellPadding` inside the` styles` function to use `spacing.unit` from current `theme`